companion/src/eeprominterface.cpp

```diff
@@ -582,14 +582,14 @@ bool RawSource::isPot() const
 {
   return (type == SOURCE_TYPE_STICK &&
           index >= CPN_MAX_STICKS &&
-          index < CPN_MAX_STICKS+getCurrentFirmware()->getCapability(Pots));
+          index < CPN_MAX_STICKS + getBoardCapability(getCurrentBoard(), Board::Pots));
```

Comment:
with a new class Board it would be `getCurrentBoard()->getCapability(Pots)` which is easy to read, would it be ok?